src/App.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
-import './App.css';
+// eslint-disable-next-line sort-imports
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import SideNavbar from 'components/SideNavbar/SideNavbar';
+import chatPage from 'pages/chatPage';
+import historyPage from 'pages/historyPage';
+import keywordPage from 'pages/keywordPage';
+import myPage from 'pages/myPage';
 
-function App() {
-  return (
-    <div>
-      Loading
-    </div>
-  );
-}
+const App = () => (
+  <div className="Main">
```

Comment:
왜 className 을 만들어주신거죠? 

```suggestion
  <div>
```

Reply:
일단 지금 테스트로만 올려두는 거라고 생각해서 나중에 App.js 수정 시에 변경하려고 임시로 만들어서 사용했습니다

Reply:
네...? 무슨 소리죠...? 뭘 테스트 하려고 하길래 `className` 사용한거죠?

Reply:
형식만 맞추려고 올렸습니다.  className 꼭 필요한 경우 아니면 사용자제하겠습니다